op-chain-ops/deployer/state/io.go

```diff
@@ -0,0 +1,57 @@
+package state
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/BurntSushi/toml"
+)
+
+func ReadJSONFile(path string, data any) error {
```

Comment:
We have similar JSON helpers in `op-service` already, thoughts on making new ones vs importing those?

Reply:
+1

Reply:
Will import these for sure if they already exist!